src/util/s3select.js

```diff
@@ -33,19 +35,50 @@ class S3SelectStream extends Transform {
         Buffer.from([0, 0, 0, 40]), //headers byte length
         Buffer.from([0xc1, 0xc6, 0x84, 0xd4]), //crc of above (end of PRELUDE)
         //DATA begins, HEADERS begin
```

Comment:
What does it mean?

Reply:
Data and Headers are parts of the AWS-encoded protocol (described in the link above class).